editor/src/components/canvas/ui-jsx-canvas.tsx

```diff
@@ -160,11 +160,15 @@ export interface CanvasReactReportErrorCallback {
   reportError: (editedFile: string, error: Error, errorInfo?: React.ErrorInfo) => void
```

Comment:
shouldn't this be deleted then?

Reply:
It's still used elsewhere (e.g. in the `CanvasErrorBoundary` itself)